scripts/hudson/narayana.sh

```diff
@@ -62,13 +63,13 @@ function init_test_options {
         export COMMENT_ON_PULL=""
         export AS_BUILD=0 AS_CLONE=0 AS_DOWNLOAD=0 AS_TESTS=0 NARAYANA_BUILD=0 NARAYANA_TESTS=0 XTS_AS_TESTS=0 XTS_TESTS=0 TXF_TESTS=0 txbridge=0
         export RTS_AS_TESTS=0 RTS_TESTS=0 JTA_CDI_TESTS=0 QA_TESTS=0 SUN_ORB=0 OPENJDK_ORB=0 JAC_ORB=0 JTA_AS_TESTS=0
-        export PERF_TESTS=0 OSGI_TESTS=0 TOMCAT_TESTS=0 LRA_TESTS=0
+        export PERF_TESTS=0 OSGI_TESTS=0 TOMCAT_TESTS=0 LRA_THORNTAIL_TESTS=0 LRA_WILDFLY_TESTS=0
```

Comment:
Similar comment as my first one, we need to migrate to quarkus since thorntail isn't getting much love anymore.

And following on from my earlier comment, having a single LRA variable which we can conditionally modify using LRA_THORNTAIL_TESTS and LRA_WILDFLY_TESTS is going help with avoiding adding complexity to this script (we're already well over thousand lines of shell).

Reply:
See previous comment

Reply:
Let's keep it to just one axis then which has the benefit that the job configs won't need to change.

Reply:
resolving